### ISSUE TITLE
Get pull requests for user endpoint

### ISSUE THREAD

**Issue by Style87** [MunifTanjim/node-bitbucket#80]
Is the /2.0/pullrequests/{selected_user} endpoint implemented?
After digging into the source the answer is definitely no. So I would like to request an enhancement to include this endpoint as either bitbucket.user.listPullRequests or bitbucket.pullrequests.listForUser (or both with an alias). Both of which would require a selected_user parameter. If I manage to learn enough TS to create this enhancement myself I'll make a PR from my fork.

**Comment by Style87** [comment]
@MunifTanjim , I'm confused as to how this resolves the issue. While a value was given in the endpoint-names.json file since no definition exists in the routes.json file the endpoint is still unable to be called. I believe something like the snippet below would have to be added to one of the main keys in routes.json.

```
    "listPullrequestsForUser": {
      "method": "GET",
      "params": {
        "state": {
          "enum": ["MERGED", "SUPERSEDED", "OPEN", "DECLINED"],
          "type": "string"
        },
        "selected_user": {
          "required": true,
          "type": "string"
        }
      },
      "returns": "PaginatedPullrequests",
      "url": "/pullrequests/{selected_user}"
    }
```
Apologies if I'm way off base. While I'm a programmer I'm not used to json configuration building into compiled code.

**Comment by MunifTanjim** [comment]
Hey @Style87 , I'm sorry about this. You are absolutely right. I forgot to run `npm run generate:routes` after I added that name.

**Comment by MunifTanjim** [comment]
It should be available in `v2.5.1` 🎉
